proto/tripcommon.proto

```diff
@@ -112,6 +112,16 @@ message TransitStationInfo {
   optional LatLng ll = 3;                // Latitude, longitude of the station
 }
 
+message BikeShareStationInfo {
+  optional string name = 1;
+  optional string ref = 2;
+  optional string capacity = 3;
```

Comment:
i believe this should also be a number. 

```suggestion
  optional uint32 capacity = 3;
```

then below you'll want to use `strtoul` and if it cant parse the string then dont set the capacity

Reply:
Fixed ! it's better this way